frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightContent.tsx

```diff
@@ -31,6 +31,10 @@ const PipelineRunDrawerRightContent: React.FC<PipelineRunDrawerRightContentProps
 
   return (
     <DrawerPanelContent
+      // This forces a re-render to solve resize of the Log viewer inside Drawer.
+      onResize={() => {
+        window.dispatchEvent(new Event('resize'));
+      }}
```

Comment:
Calculating `lineHeight `of logs in logviewer doesn't seems to work when the drawer resizes even after using `ResizeObserver` and `onsize()` with updating state. I don't think the proposed solution is ideal solution, but I 'm not sure how can we solve this issue. Please let me know If there any ways to solve this.

Reply:
This is probably good enough for now, ideally the LogViewer should be able to handle this internally. Maybe we should open an issue to follow up when https://github.com/patternfly/react-log-viewer/pull/28 is merged, or some other equivalent fix.

This workaround seems harmless to me for now, especially since `onResize` in the drawer will only trigger once when the user releases the mouse button after dragging.